HotelPriceScout/Data/Interface/SettingsManager.cs

```diff
@@ -22,6 +23,30 @@ public class SettingsManager : ComponentBase
         public bool modalStart = false;
         public bool updateYes = false;
 
+        public IEnumerable<DateTime> GetNotificationTimes()
+        {
+            List<DateTime> result = new List<DateTime>();
+            switch (notificationPickedPass)
+            {
+                case 1:
+                    result.Add(timeValuePass);
+                    break;
+                case 2:
+                    result.Add(timeValuePass);
+                    result.Add(timeValuePass2);
+                    break;
+                case 3:
+                    result.Add(timeValuePass);
+                    result.Add(timeValuePass2);
+                    result.Add(timeValuePass3);
+                    break;

```

Comment:
use fall-through here instead by switching case numbers and then removing breaks.

Reply:
of course husk en break inden default